renku/ui/cli/dataset.py

```diff
@@ -1223,3 +1223,13 @@ def mount(name, existing, unmount, yes):
         command.execute(name=name)
     else:
         command.execute(name=name, existing=existing, yes=yes)
+
+
+@dataset.command(hidden=True)
```

Comment:
Should we make these commands not hidden? We've had them for a while already.
@rokroskar ?

Reply:
I suggest we keep them hidden at least for one more sprint. We also need to add documentation for this feature.

Reply:
They should have been public yesterday! I've been using them for weeks :)